packages/mdc-typography/_mixins.scss

```diff
@@ -22,7 +22,7 @@
   -webkit-font-smoothing: antialiased;
 }
 
-@mixin mdc-typography($style) {
+@mixin mdc-typography($style, $adjust-margin: false) {
```

Comment:
Please update the signature of `mdc-typography` in README.md (under ["Sass Variables and Mixins"](https://github.com/pndewit/material-components-web/blob/86ac580c9eb8280c1622b5ea0efc0f933aaca3f4/packages/mdc-typography/README.md#sass-variables-and-mixins)) to include this new param

Reply:
done